diff -Naur linux-2.6.20-old/scripts/unifdef.c linux-2.6.20-new/scripts/unifdef.c
--- linux-2.6.20-old/scripts/unifdef.c  2007-02-04 11:44:54.000000000 -0700
+++ linux-2.6.20-new/scripts/unifdef.c  2010-12-29 17:16:28.000000000 -0700
@@ -206,7 +206,7 @@
 static void             error(const char *);
 static int              findsym(const char *);
 static void             flushline(bool);
-static Linetype         getline(void);
+static Linetype         uc_getline(void);
 static Linetype         ifeval(const char **);
 static void             ignoreoff(void);
 static void             ignoreon(void);
@@ -512,7 +512,7 @@
 
	for (;;) {
		linenum++;
-		lineval = getline();
+		lineval = uc_getline();
		trans_table[ifstate[depth]][lineval]();
		debug("process %s -> %s depth %d",
		    linetype_name[lineval],
@@ -526,7 +526,7 @@
  * help from skipcomment().
  */
 static Linetype
-getline(void)
+uc_getline(void)
 {
	const char *cp;
	int cursym;
