diff -Naur linux-2.6.34.6-old/arch/powerpc/include/asm/ptrace.h linux-2.6.34.6-new/arch/powerpc/include/asm/ptrace.h
--- linux-2.6.34.6-old/arch/powerpc/include/asm/ptrace.h    2010-08-26 16:44:38.000000000 -0700
+++ linux-2.6.34.6-new/arch/powerpc/include/asm/ptrace.h    2010-09-12 09:52:55.000000000 -0700
@@ -27,8 +27,10 @@
 #ifdef __KERNEL__
 #include <linux/types.h>
 #else
+#ifndef __ASSEMBLY__
 #include <stdint.h>
 #endif
+#endif
 
 #ifndef __ASSEMBLY__
 
